README.md

```diff
@@ -47,4 +49,6 @@ It's possible to control the chromecasts, by sending a message to the `<MQTT_ROO
 
 if you want to start playing a new media, then you can use the endpoint `<root topic>/cast/control/play`, send the URL for your media as payload
 
+if you want to set the volume, then you can use the endpoint `<MQTT_ROOT>/friendly_name/control/volume`, send a value between 0 and 1 as the payload
```

Comment:
```suggestion
if you want to set the volume, then you can use the endpoint `<MQTT_ROOT>/friendly_name/control/volume`, send a value between 0 and 100 as the payload
```

Reply:
Ah, good catch - will push an update for the docs.